packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -587,3 +595,45 @@ export class TelemetryNullLogger implements ITelemetryLogger {
     public sendErrorEvent(event: ITelemetryErrorEvent, error?: any): void {}
     public sendPerformanceEvent(event: ITelemetryPerformanceEvent, error?: any): void {}
 }
+
+/**
+ * Take in a event object, stringify any fields that are non-primitives, and return the new event object.
+ * @param event - Event with fields you want to stringify.
```

Comment:
Can you please update this comment? Just something high level, doesn't need to mention stringify (and it's just not accurate at the moment)